controllers/feeds/getOneFeedChat.js

```diff
@@ -38,17 +38,14 @@ const getReaction = (req, latest_reactions, constantActor) => {
 };
 
 const getOneFeedChatService = async (req, res) => {
-  let oppositeUserId = null;
   let selfSignedUserId = null;
   let selfAnonymousUserId = null;
 
   if (req?.user?.is_anonymous) {
-    oppositeUserId = await UsersFunction.findSignedUserId(User, req.userId);
+    selfSignedUserId = await UsersFunction.findSignedUserId(User, req.userId);
     selfAnonymousUserId = req.userId;
-    selfSignedUserId = oppositeUserId;
   } else {
     const user = await UsersFunction.findAnonymousUserId(User, req.userId);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The logic for assigning `selfSignedUserId` and `selfAnonymousUserId` seems to be reversed. In the case of an anonymous user, you are finding a signed user ID and assigning it to `selfSignedUserId`, which is correct. But in the else block (when the user is not anonymous), you are finding an anonymous user ID and assigning it to `selfAnonymousUserId`. This seems incorrect as per the variable names and their intended use. Please verify this logic.
```diff
   if (req?.user?.is_anonymous) {
    selfSignedUserId = await UsersFunction.findSignedUserId(User, req.userId);
     selfAnonymousUserId = req.userId;
   } else {
     const user = await UsersFunction.findSignedUserId(User, req.userId);
-     selfAnonymousUserId = user?.id;
+     selfSignedUserId = user?.id;
-     selfSignedUserId = req.userId;
+     selfAnonymousUserId = req.userId;
   }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->